sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -12,58 +59,65 @@ if (NOT DEFINED UNIFIED_RUNTIME_LIBRARY OR NOT DEFINED UNIFIED_RUNTIME_INCLUDE_D
   #        [UR][L0] Copy prebuilt L0 to avoid leaking shared folder path
   set(UNIFIED_RUNTIME_TAG b38855ed815ffd076bfde5e5e06170ca4f723dc1)
 
-  if ("level_zero" IN_LIST SYCL_ENABLE_PLUGINS)
-    set(UR_BUILD_ADAPTER_L0 ON)
+  if(SYCL_PI_UR_OVERRIDE_FETCH_CONTENT_REPOSITORY)
```

Comment:
Seems to be a typo here, above it is called: SYCL_PI_UR_OVERRIDE_FETCH_CONTENT_REPO

Reply:
Fixed.